porting/first-steps.rst

```diff
@@ -53,5 +53,22 @@ Install the required dependencies::
      python-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool \
      repo liblz4-tool bc
 
+Arch
+^^^^
+
+If you have a pure ``amd64`` running you need to add ``[multilib]`` repository to your ``/etc/pacman.conf`` . This will allow you to install and run ``i686`` packages. Please refer to `<https://wiki.archlinux.org/index.php/multilib>`_
+Assuming you have already installed ``base-devel`` group; if not, then please install ``base-devel``.
+   
+You need to install a GPG key which is required to install ``halium-devel`` meta package dependencies (``ncurses5-compat-libs`` and ``lib32-ncurses5-compat-libs``)::
+
+   gpg --recv-keys 702353E0F7E48EDB
```

Comment:
One last thing :P 

I would prefer if gpg key instruction is removed from here.. mainly because gpg key might change at one point, leaving documentation out-of-date and something we can't control.. just say install this package.. archlinux users should know about gpg key stuff already..

Reply:
probably better leave a link to arch wiki https://wiki.archlinux.org/index.php/GnuPG